x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -123,9 +149,12 @@ const BENCHMARKS_TABLE_COLUMNS: Array<EuiBasicTableColumn<Benchmark>> = [
     truncateText: true,
     width: '17.5%',
     sortable: true,
-    render: (benchmarkId: BenchmarksCisId) => {
-      return getBenchmarkCisName(benchmarkId);
-    },
+    render: (benchmarkId: Benchmark['id'], benchmark: Benchmark) => (
+      <BenchmarkButtonLink
+        benchmarkId={benchmarkId || ''}
+        benchmarkVersion={benchmark.version || ''}
+      />
+    ),
```

Comment:
could this render work with benchmark only?

```jsx
render: (benchmark: Benchmark) => (
      <BenchmarkButtonLink
        benchmarkId={benchmark.id || ''}
        benchmarkVersion={benchmark.version || ''}
      />
    )
```

Reply:
nope :(